server/Entities/Models/Author.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Entities.Models
+{
+    public class Author
+    {
+        public int AuthorId { get; set; }
+        public string AuthorName { get; set; }
```

Comment:
1. it is entity Author, it is obvious that properties belongs Author, so just Id and Name
2. do you store first name and last name together?
3. what about database validation?

Reply:
actually, be ready, that we'll have users and it should be connected with users

Reply:
done, changed naming and splited up AuthorName